src/CarwowTheme/Filters.elm

```diff
@@ -124,21 +159,26 @@ getFilterGroupItemLabel item =
 
 {-| Placeholder
 -}
-filterGroup : List (FilterGroupItem msg) -> String -> List (Html.Html msg)
-filterGroup items label =
-    List.map (\item -> filterGroupItem item label) items
+filterGroup : List (FilterGroupItem msg) -> String -> String -> List (Html.Html msg)
+filterGroup items label filterPrefix =
+    List.map (\item -> filterGroupItem item label filterPrefix) items
```

Comment:
If `item` was the final argument to `filterGroupItem`, you could do `List.map (filterGroupItem label filterPrefix)`.

And if `items` was the last argument to `filterGroup`, you could define this whole thing as
```elm
filterGroup label filterPrefix =
    List.map (filterGroupItem label filterPrefix)
```